src/app/ui/main_window.cpp

```diff
@@ -90,8 +90,66 @@ MainWindow::MainWindow()
   , m_devConsoleView(nullptr)
 #endif
 {
+}
+
+MainWindow::~MainWindow()
```

Comment:
Probably there is no need to move the ~MainWindow code to this place to keep diff to a minimum.